app/src/views/Confirm.vue

```diff
@@ -329,6 +416,25 @@ export default {
     showGasPrice(val) {
       return `Fee: $ ${significantDigits(parseFloat(this.txFees).toFixed(3))}`
     },
+    getGasDisplayString(speed, fastGasPrice) {
+      const currencyMultiplier = this.getCurrencyMultiplier
+      const ethFee = this.gasEstimate * fastGasPrice * 10 ** -9
+      const currencyFee = ethFee * currencyMultiplier
+      return `${significantDigits(currencyFee)} ${this.$store.state.selectedCurrency}`
+    },
+    getSelectedSpeed() {
+      let selectedType = ''
+      let nearest = 1000
+      const speedOptions = [...this.speedOptions]
```

Comment:
not necessary. .reverse() returns a new array and doesn't mutate original

Reply:
In case there are options with same gasPrice then I'll get the one with faster speed. I didn't intend to mutate the original so the layout would stay the same..